Conformity/Integration/wellarchitected-sync/README.md

```diff
@@ -0,0 +1,42 @@
+# Well-Architected Conformity Sync
+
+This tools sets up a synchronization link between Cloud One Conformity and the AWS Well Architected tool.
+
+The sync goes in one direction from Conformity to the AWS Well Architected tool. If configurations are remediated you need to run the synchronization again to update the Well Architected tool but keep in mind that the notes will be over-written.
+
+Setting up the Sync tool creates 2 IAM Roles and their policies, AWS SSM Parameters, a secret and a Lambda function.
+
+See https://cloudone.trendmicro.com/docs/conformity/aws-integration/#aws-well-architected-tool for further details about Conformity's integration with the Well Architected tool.
+
+See https://cloudone.trendmicro.com/docs/conformity/api-reference/tag/Well-Architected-Tool for further details about the Conformity WellArchitected sync API.
+
+## Pre-requisites
+
+- Administrator access to the AWS Console. In default, sufficient access rights to run CloudFormation templates and to invoke Lambda functions from the AWS CLI or AWS CloudShell
+- A workload configured in the AWS Well Architected tool
```

Comment:
Link to AWS Doc how to do this successfully

Reply:
Fixed.